src/character.cpp

```diff
@@ -6432,7 +6428,7 @@ void Character::set_cardio_acc( int ncardio_acc )
 
 void Character::reset_cardio_acc()
 {
-    set_cardio_acc( base_bmr() / 2 );
+    set_cardio_acc( 1000 );
```

Comment:
probably put that value into a constant to avoid magic numbers

Reply:
Where should I define the constant? In game_constants.h?

Reply:
Or is a function get_cardio_acc_base() that returns 1000 good enough?